src/panel/poi/PoiPanel.jsx

```diff
@@ -278,18 +278,17 @@ export default class PoiPanel extends React.Component {
             <div className="poi_panel__fullContent">
               <PoiBlockContainer poi={poi} covid19Enabled={covid19Enabled} />
               {isFromPagesJaunes(poi) &&
-              <div className="poi_panel__info-partnership u-text--caption">
+              <div className="poi_panel__info-partnership u-text--caption u-mb-s">
                 {_('In partnership with')}
                 <img src="./statics/images/pj.svg" alt="PagesJaunes" width="80" height="16" />
               </div>
               }
-              {poi.id.match(/latlon:/) && <div className="service_panel__categories--poi">
-                <h3 className="u-text--smallTitle">
-                  {_('Search around this place', 'poi')}
-                </h3>
-                <CategoryList />
-              </div>}
               {isFromOSM(poi) && <OsmContribution poi={poi} />}
+              <Divider paddingTop={0} className="poi_panel__fullWidthDivider" />
```

Comment:
Would it be possible to create two blocks with class `poi_panel__fullContent`, and use a divider between them ? This way we would not need to set negative margin on divider.

Reply:
I didn't want to do that in this PR before we choose a generic box layout system for the rest of the app, so the negative margin approach seemed simple.
And I don't want to touch `.poi_panel__fullContent` or duplicate it, as it's used for the mobile Panel to hide content dynamically. What I would need here is divs with only horizontal paddings.
If you prefer this solution I can still do it for this panel only, even if it touches little things outside.

Reply:
seems fair, we will do this later